depends/packages/libXau.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=fdd477320aeb5cdd67272838722d6b7d544887dfe7de46e1e7cc0c27c
 $(package)_dependencies=xproto
 
 define $(package)_set_vars
-  $(package)_config_opts=--disable-shared
+  $(package)_config_opts=--disable-shared --disable-lint-library --without-lint
```

Comment:
Nit: Took a look at `--disable-xthreads` in the autoconf files... It seems that it's currently enabled. I'm not sure what the impact is of disabling it so we'll leave it for now.

Reply:
Yeah a comment would be great for the future!

Reply:
Added a comment.